.github/workflows/prod-deploy.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           echo "Restore"
           dotnet restore $SERVICE_NAME.csproj
           echo "Clean"
-          dotnet clean $SERVICE_NAME.csproj -c "Release" -o obj/Azure/publish
```

Comment:
I don't think we should be changing this to Docker since we are not using Docker in this workflow

Reply:
Updated path to `obj/build-output/publish`.